utils/appliance/__init__.py

```diff
@@ -2210,9 +2213,16 @@ def __get__(self, instance, owner):
         return get_or_create_current_appliance()
 
 
+class CurrentContext(object):
+    def __get__(self, instance, owner):
+        return get_or_create_current_appliance().context()
+
+
 class Navigatable(object):
 
     appliance = CurrentAppliance()
+    parent = CurrentContext()
```

Comment:
This one will probably go now that we are down the road with collections objects.